frontend/src/AlertConfiguration/AlertConfigurationComponents/CreateAlertConfiguration.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import Avatar from '@material-ui/core/Avatar';
```

Comment:
I noted this in the overall comment but I think this file especially has a lot of unnecessary imports - be sure to take them out!